tox.ini

```diff
@@ -4,10 +4,11 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py35, py36, py37
+envlist = py36, py37
```

Comment:
❔ Should we also update the list of supported Python versions in `README.md` and `docs`?

Reply:
absolutely, I forgot about it.